libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/SQLiteDatabaseConnection.m

```diff
@@ -0,0 +1,121 @@
+//
+//  SQLiteDatabaseConnection.m
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "SQLiteDatabaseConnection.h"
+
+@implementation SQLiteDatabaseConnection {
+@private
+  NSString *_databasePath;
+  sqlite3 *_DB;
+  sqlite3_stmt *_insertStatement;
+}
+
+static NSString *const insertPreparedStatement = @"INSERT INTO geocodingPairs%@"
+                                                 @"( "
+                                                 @"NATIONALNUMBER, "
+                                                 @"DESCRIPTION"
+                                                 @")"
+                                                 @"VALUES "
+                                                 @"("
+                                                 @"?,"
+                                                 @"?"
+                                                 @")";
```

Comment:
nit: This can be formatted a bit better by grouping things like the VALUES (?, ?) and the (NATIONALNUMBER, DESCRIPTION) on the same line

Reply:
fixed.